features/admin/payment/managing_payments/browsing_payments.feature

```diff
@@ -12,21 +12,37 @@ Feature: Browsing payments
         And there is an "#00000001" order with "Apple" product
         And I am logged in as an administrator
 
-    @todo @ui @api
+    @ui @api
     Scenario: Browsing payments and their states
         When I browse payments
         Then I should see a single payment in the list
         And the payment of the "#00000001" order should be "New" for "amba@fatima.org"
 
-    @todo @ui @api
+    @ui @api
     Scenario: Not seeing payments in cart state
         Given the customer "customer@example.com" added "Apple" product to the cart
         When I browse payments
         Then I should see a single payment in the list
 
-    @todo @ui @api
+    @ui @api
     Scenario: Payments are sorted by newest as default
         Given there is an "#00000002" order with "Apple" product ordered later
         When I browse payments
         Then I should see payment for the "#00000002" order as 1st in the list
         And I should see payment for the "#00000001" order as 2nd in the list
+
+    @ui @api-todo
+    Scenario: Sorting payments by date in ascending order
+        Given there is an "#00000002" order with "Apple" product ordered later
+        When I browse payments
+        And I sort payments by date in ascending order
+        Then I should see payment for the "#00000001" order as 1st in the list
+        And I should see payment for the "#00000002" order as 2nd in the list
+
+    @ui @api-todo
+    Scenario: Sorting payments by date in descending order again
+        Given there is an "#00000002" order with "Apple" product ordered later
+        When I browse payments
+        And I sort payments by date in descending order
+        Then I should see payment for the "#00000002" order as 1st in the list
+        And I should see payment for the "#00000001" order as 2nd in the list
```

Comment:
These new scenarios for sorting can be extracted into a separate file

Reply:
I've extracted these tests